<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>685</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="delits">
     <attribute name="title">
      <string>Delits</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>541</width>
          <height>461</height>
         </rect>
        </property>
        <property name="title">
         <string>Delit</string>
        </property>
        <widget class="QSplitter" name="splitter_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>481</width>
           <height>361</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Id délit</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="ledelitid"/>
         </widget>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Date de delit</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEditdelit"/>
         </widget>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_typedelit"/>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_desc_delit"/>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>250</y>
          <width>171</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_16">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_delit1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>831</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>331</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Liste des delits</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="supprimerPb_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>611</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <property name="spacing">
          <number>200</number>
         </property>
         <item>
          <widget class="QLabel" name="supprimerlbl_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Identifiant de delit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_26">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_15">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>40</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_delit2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>100</y>
          <width>431</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_27">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>40</y>
          <width>266</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="chercherlbl_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>40</y>
          <width>140</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>450</y>
          <width>431</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>401</width>
          <height>441</height>
         </rect>
        </property>
        <property name="title">
         <string>Delit</string>
        </property>
        <widget class="QSplitter" name="splitter_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>361</width>
           <height>341</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Id délit</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_mid"/>
         </widget>
         <widget class="QSplitter" name="splitter_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>Date de delit</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit_3"/>
         </widget>
         <widget class="QSplitter" name="splitter_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_12"/>
         </widget>
         <widget class="QSplitter" name="splitter_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_11"/>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="criminels">
     <attribute name="title">
      <string>Criminels</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>881</width>
        <height>591</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>200</y>
          <width>231</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>541</width>
          <height>531</height>
         </rect>
        </property>
        <property name="title">
         <string>Criminel</string>
        </property>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>501</width>
           <height>481</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>95</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Identifiant unique</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>210</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>185</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Prenom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>218</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Age</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Id delits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="supprimerPb">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>180</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>611</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>200</number>
         </property>
         <item>
          <widget class="QLabel" name="supprimerlbl">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Identifiant Unique</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_18">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Rechercher</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>761</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>361</width>
          <height>51</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_10"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>50</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>501</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>470</y>
          <width>431</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>341</width>
          <height>431</height>
         </rect>
        </property>
        <property name="title">
         <string>Nouveau données</string>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>321</width>
           <height>361</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Identifiant unique</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>95</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Prenom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_9"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>122</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>129</number>
            </property>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Age</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>411</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="chercherlbl">
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>31</y>
          <width>771</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>480</y>
          <width>411</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Nombre total de criminels arrêtés</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_17"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>460</y>
          <width>331</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Affichage d'une charte graphique</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
